docs/tests/README.md

```diff
@@ -78,17 +79,17 @@ describe('Film Card', () => {
 
 ## Mocking
 
-Le **mocking** consiste à isoler le sujet de test en remplaçant par des simulacres toutes les briques extérieures avec lesquelles il interagit. Ainsi, en cas d'échec du test, on s'assure que le problème vient bien de la fonction testée et non d'un composant externe (réseau, bases de données, lib tierce etc.)
+The **mocking** consists of isolating the test subject by replacing with a simulator all the external bricks with which he interacts. Thus, in case of failure of the test, it is ensured that the problem comes from the tested function and not from an external component (network, databases, third party, etc.).
```

Comment:
by replacing **it with a mockup that simulate all the external bricks it interacts with**.